src/tests.rs

```diff
@@ -7,7 +9,8 @@ async fn test_lambda() {
   let request = lambda_http::request::from_str(input)
     .expect("failed to create request");
 
-  let response = function_handler(request).await.expect("failed to handle request");
+  let mock_ddb_client = MockClient::new();
```

Comment:
how do I do this? I saw the [mockall](https://docs.rs/mockall/latest/mockall/) library but not sure it's the right approach.